docs/units/index.rst

```diff
@@ -252,6 +253,7 @@ Reference/API
 .. automodapi:: astropy.units.physical
 
 .. automodapi:: astropy.units.equivalencies
+   :skip: with_H0
```

Comment:
And this one too?

Reply:
How did those sneak in there?!? I used a ``fixup!`` commit to prune them from the whole history

Reply:
Gone now.